spark-jobs/src/main/scala/za/co/absa/enceladus/common/CommonJobExecution.scala

```diff
@@ -85,6 +85,13 @@ trait CommonJobExecution extends ProjectMetadata {
 
     dao.authenticate()
     val dataset = dao.getDataset(cmd.datasetName, cmd.datasetVersion)
+    val dsValidation = dao.getDatasetPropertiesValidation(cmd.datasetName, cmd.datasetVersion)
```

Comment:
I would add this part into an implicit function

Reply:
@benedeki, I will try to do it at once, I understand the usecase you are hinting.
@AdrianOlosutean - what you mean? What exactly would you apply implicitly, here? Do you mean like `dataset.validation`?

Reply:
Redone to a single call. Added a unit test and extended integ test for this, too. Hope you find it satisfactory.

@AdrianOlosutean, I have not addressed your comment because of the reasons above - is it still relevant, though?

Reply:
That was mainly an idea. If it's one call now, it's alright